src/core/core-adapter.js

```diff
@@ -268,7 +267,7 @@ export default class CoreAdapter {
     this.globalStorage.set(StorageKeys.INTENTS, new SearchIntents({}));
     this.globalStorage.set(StorageKeys.NAVIGATION, new Navigation());
     this.globalStorage.set(StorageKeys.ALTERNATIVE_VERTICALS, new AlternativeVerticals({}));
-    this.globalStorage.set(StorageKeys.DIRECT_ANSWER, new DirectAnswer({}));
+    this.globalStorage.set(StorageKeys.DIRECT_ANSWER, {});
```

Comment:
ah does the direct answer still reset with backnav whether it's `new DirectAnswer({})` or `{}` here? just want to be extra careful with back nav behavior

Reply:
I was experimenting with something and then I reverted, so this should actually just be `new DirectAnswer({})`